### PR TITLE
Reduce memory leak and fix too many connection issue during tests

### PR DESCRIPTION
**Pull request by B-Galati**

### CODE REVIEW

**Comment by thomasbisignani** [comment]
:fire: :medal_sports:

File: tests/CitizenProject/CitizenProjectManagerTest.php [hunk 1]
```diff
@@ -101,10 +100,11 @@ protected function setUp()
 
     protected function tearDown()
     {
-        $this->loadFixtures([]);
+        $this->cleanupContainer($this->container);
```
Comment (Remg) [inline tests/CitizenProject/CitizenProjectManagerTest.php]: could you call of `ControllerTestTrait::kill()` here ?

File: tests/Entity/AbstractAdherentTokenTest.php [hunk 2]
```diff
@@ -73,4 +73,14 @@ private function generateToken($adherent, string $lifetime = '+1 day')
     {
         return call_user_func([$this->tokenClass, 'generate'], $adherent, $lifetime);
     }
+
+    protected function tearDown()
+    {
+        $this->cleanupContainer($this->container);
```
Comment (Remg) [inline tests/Entity/AbstractAdherentTokenTest.php]: same here ?

File: tests/Mailer/Message/AdherentTerminateMembershipMessageTest.php [hunk 3]
```diff
@@ -33,4 +33,14 @@ public function testCreateAdherentTerminateMembershipMessage()
         $this->assertSame('Kévin CORNIL', $recipient->getFullName());
         $this->assertSame(['target_firstname' => 'Kévin'], $recipient->getVars());
     }
+
+    protected function tearDown()
+    {
+        $this->cleanupContainer($this->container);
```
Comment (Remg) [inline tests/Mailer/Message/AdherentTerminateMembershipMessageTest.php]: same here

**Comment by B-Galati** [comment]
Thank you all :-)

File: tests/Entity/AbstractAdherentTokenTest.php (hunk 2 above)
Comment (B-Galati) [reply tests/Entity/AbstractAdherentTokenTest.php]: done

File: tests/Mailer/Message/AdherentTerminateMembershipMessageTest.php (hunk 3 above)
Comment (B-Galati) [reply tests/Mailer/Message/AdherentTerminateMembershipMessageTest.php]: done

File: tests/CitizenProject/CitizenProjectManagerTest.php (hunk 1 above)
Comment (B-Galati) [reply tests/CitizenProject/CitizenProjectManagerTest.php]: done

**Pull request merged**